problem-gson/src/test/java/org/zalando/problem/MyProblem.java

```diff
@@ -0,0 +1,87 @@
+package org.zalando.problem;
+
+import com.google.gson.annotations.SerializedName;
+
+import javax.annotation.Nullable;
+import java.net.URI;
+
+class MyProblem implements Problem {
+
+    @SerializedName("type")
+    private final URI myType;
+
+    @SerializedName("title")
+    private final String myTitle;
+
+    @SerializedName("status")
+    private final StatusType myStatus;
+
+    @SerializedName("detail")
+    private final String myDetail;
+
+    @SerializedName("instance")
+    private final URI myInstance;
+
+    private String misc;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'misc' could be replaced by a local variable.](https://app.codacy.com/app/whiskeysierra/problem/pullRequest?prid=4476243)